test/e2e/kubelet_stats.go

```diff
@@ -157,8 +157,8 @@ func getContainerInfo(c *client.Client, nodeName string, req *kubelet.StatsReque
 		return nil, err
 	}
 	data, err := c.Post().
-		Prefix("proxy").
 		Resource("nodes").
+		SubResource("proxy").
```

Comment:
same goes for all the e2e tests, the subresource tests need to be version guarded somehow